docs/contracts/libraries.rst

```diff
@@ -184,24 +184,12 @@ custom types without the overhead of external function calls:
 It is possible to obtain the address of a library by converting
 the library type to the ``address`` type, i.e. using ``address(LibraryName)``.
 
-As the compiler cannot know where the library will be
-deployed at, these addresses have to be filled into the
-final bytecode by a linker
-(see :ref:`commandline-compiler` for how to use the
-commandline compiler for linking). If the addresses are not
-given as arguments to the compiler, the compiled hex code
-will contain placeholders of the form ``__Set______`` (where
-``Set`` is the name of the library). The address can be filled
-manually by replacing all those 40 symbols by the hex
-encoding of the address of the library contract.
-
-.. note::
-    Manually linking libraries on the generated bytecode is discouraged, because
-    in this way, the library name is restricted to 36 characters.
-    You should ask the compiler to link the libraries at the time
-    a contract is compiled by either using
-    the ``--libraries`` option of ``solc`` or the ``libraries`` key if you use
-    the standard-JSON interface to the compiler.
+As the compiler does not know where the library will be deployed at, the compiled hex code will
+contain placeholders of the form ``__$53aea86b7d70b31448b230b20ae141a537$__``. Such bytecode is
```

Comment:
I think it's not clear here what the hex inside `__$...$__` is

Reply:
Done. But note that it's also explained in the section about the linker. I wanted to keep it short here and just redirect the user there if he's curious about the details.